File: CarShop_Mobile/app/src/main/java/com/example/carshop_mobile/main/Model/Appointment.kt
package com.example.carshop_mobile.main.Model

import com.google.gson.annotations.SerializedName
import java.util.*

enum class State{
    @SerializedName("0")
    NotFinished,
    @SerializedName("1")
    Working,
    @SerializedName("2")
    Finished
}

class Appointment() {
    @SerializedName("idAppointment")
    var idAppointemnt:Int? = null
    @SerializedName("date")
    var date : String = ""
    @SerializedName("time")
    var time: String = ""
    @SerializedName("user")
    var user:UserMobile? = null
    @SerializedName("customerName")
    var customerName:String = ""
    @SerializedName("customerPhone")
    var customerPhone:String = ""
    @SerializedName("state")
    var state:State = State.NotFinished


    constructor(date:String,time:String,currentUser:UserMobile,name:String,phone:String,state:State) : this()
    {
        this.date = date
        this.time = time
        this.user = currentUser
        this.customerName = name
        this.customerPhone = phone
        this.state = state
    }
}

File: CarShop_REST_API/CarShop_REST_API/Model/Appointment.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace CarShop_REST_API.Model
{
    public enum State
    {
        NotFinished,
        Working,
        Finished
    }
    public class Appointment
    {
        [Key]
        public int IDAppointment { get; set; }
        public string Date { get; set; }
        public string Time { get; set; }
        public UserMobile User { get; set; }
        public string CustomerName { get; set; }
        public string CustomerPhone { get; set; }
        public State State { get; set; }

    }
}
